src/key_values.rs

```diff
@@ -0,0 +1,10 @@
+/// The identifier for the configuration that this Omni-Paxos replica is part of.
```

Comment:
Let's not have this file under `/src`. Can you create a new module `utils` and put it there? Also the filename can be more clear.

I think something like:  `omnipaxos/utils/hocon_kv.rs` would be good.